le_pr_delivery/delivery/urls.py

```diff
@@ -1,11 +1,14 @@
 from django.conf.urls import patterns, include, url
 from django.contrib import admin
 from . import views
+from delivery.views import *
```

Comment:
My bad, I wasn't clear in the previous message.
``` from . import views ``` will import the file name but not its contents i.e home(), done() etc. Thus, we need that along with ``` from delivery.views import * ```
Alternative to this would be to use ``` admin.autodiscover() ``` in views, but I don't it since we won't be using ```views.py``` in any other files. 
So keeping this file as it is